hardhat.config.ts

```diff
@@ -4,6 +4,7 @@ import '@nomiclabs/hardhat-ethers'
 import '@openzeppelin/hardhat-upgrades'
 import '@nomiclabs/hardhat-waffle'
 import {log} from './config/logging'
+import '@nomiclabs/hardhat-web3'
```

Comment:
question: what in the web3 dependency do you need?

_Now we're using both web3 and ethers_

Reply:
I added web3-utils because is has the right packing+hash function to test solidity packing+hash for the tree leafs.

Reply:
removed -> import '@nomiclabs/hardhat-web3'